src/message.js

```diff
@@ -479,16 +479,18 @@ export class Message {
   }
 
   /**
-   * Compresses the message (the literal and -if signed- signature data packets of the message)
+   * Compresses the message (the literal and -if signed ß- signature data packets of the message)
```

Comment:
I see you're starting to integrate into German, inserting random German ~sentences~ letters in your work :upside_down_face: